packages/preview2-shim/lib/browser-async/cli.js

```
@@ -0,0 +1,136 @@
+const textDecoder = new TextDecoder();
+
+let stdinStream, stdoutStream, stderrStream;
+let _env = [], _args = [], _cwd = "/";
+export function _setEnv (envObj) {
+  _env = Object.entries(envObj);
+}
+export function _setArgs (args) {
+  _args = args;
+}
+export function _setCwd (cwd) {
+  fsSetCwd(_cwd = cwd);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)